docs/contribute/api-docs-contribution/guidelines.md

```diff
@@ -0,0 +1,575 @@
+- **Provide usage guidance:** Explain when to use each option and any performance implications
+- **Document special behavior:** Note if some values are deprecated, experimental, or have unique characteristics
+- **Keep it concise:** Use brief, focused descriptions that explain the essential differences
+
+:::{tip}
+You can skip documenting enum values that are self-explanatory (like `true`/`false`) or follow standard conventions (like HTTP status codes).
+:::
+
+### Examples
+
+::::{tab-set}
+:group: implementations
+
+:::{tab-item} OpenAPI
+:sync: openapi
+
+In OpenAPI specifications, enum descriptions are documented in schema definitions:
+
+```yaml
+components:
+  schemas:
+    SearchType:
+      type: string
+      enum:
+        - query_then_fetch
+        - dfs_query_then_fetch
+        - query_and_fetch
+      description: |
+        The search execution strategy:
+        - `query_then_fetch`: Searches across all fields using default settings
+        - `dfs_query_then_fetch`: Performs distributed frequency scoring before fetching results  
+        - `query_and_fetch`: Fetches results immediately without scoring optimization
+```
+
+For inline enums in parameters:
+
+```yaml
+parameters:
+  - name: refresh
+    in: query
+    schema:
+      type: string
+      enum: [true, false, wait_for]
+      description: |
+        When to refresh the affected shards:
+        - `true`: Refresh immediately after the operation
+        - `false`: Don't refresh (default)
+        - `wait_for`: Wait for a refresh to happen naturally
+```
+:::
+
+:::{tab-item} Elasticsearch
+:sync: elasticsearch
+
+In TypeScript definitions, enum descriptions are added as JSDoc comments above each value:
+
+```ts
+export enum SearchType {
+  /** Searches across all fields using default settings. */
+  query_then_fetch = "query_then_fetch",
+  
+  /** Performs distributed frequency scoring before fetching results. */
+  dfs_query_then_fetch = "dfs_query_then_fetch",
+  
+  /** Fetches results immediately without scoring optimization. */
+  query_and_fetch = "query_and_fetch"
+}
+```
+
+The compiler automatically extracts these descriptions and formats them as lists in the property documentation:
+
+- `query_then_fetch`: Searches across all fields using default settings
+- `dfs_query_then_fetch`: Performs distributed frequency scoring before fetching results
+- `query_and_fetch`: Fetches results immediately without scoring optimization
+:::
+
+::::
+
+
+## Add examples
+
+Examples help users understand how to use your API with realistic request and response data. Well-written descriptions transform examples from code snippets into effective learning tools.
+
+Here are some principles for effective examples:
+
+- **Use realistic data:** Provide examples that reflect actual use cases rather than placeholder values
+- **Write clear summaries:** Introduce the example with a meaningful summary that explains its purpose
```

Comment:
The summary is currently used as the label in the drop-down selector, so it needs to be very brief
```suggestion
- **Write clear summaries:** Introduce the example with a brief summary that explains its purpose
```

Reply:
I'll add parenthesis about why needs to be very brief